auditbeat/Jenkinsfile.yml

```
@@ -19,6 +18,11 @@ stages:
           make -C auditbeat check;
           make -C auditbeat update;
           make check-no-changes;
+    Lint-Xpack:
+        make: |
+          make -C x-pack/auditbeat check;
+          make -C x-pack/auditbeat update;
+          make check-no-changes;
```

Could we join these stages in a single one? (Same in the rest of beats)
```suggestion
          make -C auditbeat check;
          make -C auditbeat update;
          make -C x-pack/auditbeat check;
          make -C x-pack/auditbeat update;
          make check-no-changes;
```

---

That was also another approach, let me do it